pkg/notes/document/testdata/document_without_downloads.md.golden

```diff
@@ -0,0 +1,44 @@
+# Release notes for v1.16.1
+
+[Documentation](https://docs.k8s.io/docs/home)
+
+# Changelog since v1.16.0
+
+## Urgent Upgrade Notes 
+
+### (No, really, you MUST read this before you upgrade)
+
+ - If an API changes and no one documented it, did it really happen?
+ 
+## Changes by Kind
+
+### Other (Bug, Cleanup or Flake)
+ - This should definitely get you promoted.
+ 
+### API Change
+ - This might make people sad...or happy.
+ 
+### Bug
+ - This will likely get you promoted.
+ 
+### Cleanup
+ - This usually does not get you promoted but it should.
+ 
+### Deprecation
+ - Delorted.
+ 
+### Design
+ - Change the world.
+ 
+### Documentation
+ - There was  a library in Alexandria, Egypt once.
+ 
+### Failing Test
+ - Please run presubmit test!
+ 
+### Feature
+ - This will get you promoted.
+ 
+### Flake
```

Comment:
The notes by kind should be sorted by their priority.

Reply:
This is how I thought about solving for sorting. It's kinda a challenge to sort currently since the template is given a map. We need the `kind` information for rendering but to preserve order we need a slice (i.e. we loose kind information):

https://github.com/kubernetes/release/pull/1154

I thought about other ways to solve this in the template but I wanted to keep the template deadly simple and avoid putting logic (like sorting) in there.
Another way I thought of was just to have a function

Reply:
See my comments there :)

Reply:
Okay sgtm. I'll implement that PR, submit, and then submit this.

Reply:
/hold

For https://github.com/kubernetes/release/pull/1154